Languages ready/French.js

```diff
@@ -213,7 +213,7 @@ export default {
         Database_Delete_Title_7: "Mot de passe",
         Database_Delete_Title_8: "jdbcConnectionString",
         Database_Rotate_Button: "Rotate Password",
-        Database_Delete_Desc_1: `La suppression d'une base de données est une action permanente, elle ne peut pas être annulée.Cela supprimera définitivement la base de données "{{database}}" et supprimera toutes les données associées.`,
+        Database_Delete_Desc_1: "La suppression d'une base de données est une action permanente, elle ne peut pas être annulée. Cela supprimera définitivement la base de données "{{database}}" et supprimera toutes les données associées.",

```

Comment:
Keep this whole string between **``**, otherwise it will conflict with "{database}"